cl/recap/tasks.py

```diff
@@ -115,7 +116,7 @@ async def process_recap_upload(pq: ProcessingQueue) -> None:
         docket = await process_recap_docket(pq.pk)
         await sync_to_async(add_or_update_recap_docket.delay)(docket)
     elif pq.upload_type == UPLOAD_TYPE.ATTACHMENT_PAGE:
-        await process_recap_attachment(pq.pk)
+        await look_for_doppelganger_rds_and_process_recap_attachment(pq.pk)
```

Comment:
I think you should do two lines of logic here so that you wind up with:

```
subdocket_att_page_pks = find_subdocket_att_page_rds(...)
for pk in subdocket_att_page_pks:
    process_recap_attachment(pk)
```

A couple things here:

1. I'd rather not have the word `doppelganger` find its way into our codebase. It has become slang around here, but we can be more precise. Let's try "subdocket" instead.

2. The approach above separates out the two steps and allows you to merge your two lookup routines into one such that `look_for_doppelganger_rds_and_process_recap_attachment` and `look_for_doppelganger_rds` can just be one function, `find_subdocket_att_page_rds`. I think this should be simpler and cleaner than:
   - look for doppel and process
   - find doppel rds
   - for each found item, process recap attachment
   - then, within that, merge_attachment_page_data

   Instead, we have: find subdockets; for each, process it (which, yeah, still calls the merge function).

    I think this should be simpler?